{"name": "使用无效凭据登录", "status": "failed", "statusDetails": {"message": "AssertionError: 错误提示内容不符，实际为: \nassert '用户名或密码错误' in ''", "trace": "self = <flutter_app_test.testcases.test_login.TestLogin object at 0x0000018C87CB2C10>\n\n    @allure.story(\"异常登录\")\n    @allure.title(\"使用无效凭据登录\")\n    def test_invalid_login(self):\n        \"\"\"测试使用无效的用户名和密码登录\"\"\"\n        login_page = LoginPage(self.driver)\n    \n        with allure.step(\"输入无效的用户名和密码\"):\n            login_page.login(\"invalid_user\", \"invalid_password\")\n    \n        with allure.step(\"验证错误信息显示\"):\n            error_message = login_page.get_error_message()\n>           assert \"用户名或密码错误\" in error_message, f\"错误提示内容不符，实际为: {error_message}\"\nE           AssertionError: 错误提示内容不符，实际为: \nE           assert '用户名或密码错误' in ''\n\ntestcases\\test_login.py:24: AssertionError"}, "description": "测试使用无效的用户名和密码登录", "steps": [{"name": "输入无效的用户名和密码", "status": "passed", "start": 1748521513347, "stop": 1748521519758}, {"name": "验证错误信息显示", "status": "failed", "statusDetails": {"message": "AssertionError: 错误提示内容不符，实际为: \nassert '用户名或密码错误' in ''\n", "trace": "  File \"C:\\项目文件\\localAll\\flutter_app_test\\testcases\\test_login.py\", line 24, in test_invalid_login\n    assert \"用户名或密码错误\" in error_message, f\"错误提示内容不符，实际为: {error_message}\"\n"}, "start": 1748521519758, "stop": 1748521521505}], "start": 1748521513347, "stop": 1748521521506, "uuid": "cddcf303-a86a-4802-a9f7-c4e8f4907622", "historyId": "f85285519da8525fb5620425bca75abe", "testCaseId": "f85285519da8525fb5620425bca75abe", "fullName": "testcases.test_login.TestLogin#test_invalid_login", "labels": [{"name": "feature", "value": "登录功能"}, {"name": "story", "value": "异常登录"}, {"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-QVK4EQ1"}, {"name": "thread", "value": "16076-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_login"}]}